specification/common-types/resource-management/v3/types.json

```diff
@@ -102,6 +102,35 @@
       "description": "The resource model definition containing the full set of allowed properties for a resource. Except properties bag, there cannot be a top level property outside of this set.",
       "type": "object",
       "properties": {
+        "id": {
```

Comment:
From my understanding, this should not be included.

Reply:
Yes, common-types/ changes need to removed from this PR.

This is the PR made common-types changes that need to sync to your ADO repo.
https://github.com/Azure/azure-rest-api-specs/pull/18789